Code/Searchor/__init__.py

```diff
@@ -98,3 +98,8 @@ def SearchQuora(query):
   query = query.replace(" ", "%20")
   query_URL = "https://www.quora.com/search?q=" + query
   return query_URL
+
+def SearchUrbanDictionary(query):
+  query = query.replace(" ", "+")
```

Comment:
Isn't it supposed to be %20 instead of + ?

Reply:
My bad! I will do that.